### ISSUE TITLE
.scss files from asciidoctor fail

### ISSUE THREAD

**Issue by edbratt** [jakartaee/jekyll-theme-jakarta-ee#15]
**Describe the bug**
We use asciidoctor for our product guides and manuals. These include .scss files. These seem to cause the template to bomb out, resulting in failure to generate the web-site.
This file: "print.scss"
```
---
---

@import "{{ site.theme }}";
h1 {
  font-size: 1.8em;
}
```
Generates the following error:
The page build failed for the `gh-pages` branch with the following error:

Your SCSS file `guides/mq-dev-guide-jmx/assets/css/print.scss` has an error on line 1: File to import not found or unreadable: jekyll-theme-jakarta-ee. Load path: /hoosegow/.bundle/ruby/2.5.0/gems/jekyll-theme-primer-0.5.3/_sass. For more information, see https://help.github.com/articles/page-build-failed-invalid-sass-or-scss/.

**To Reproduce**
you can see a sample repository at: https://github.com/edbratt/mq-site-test

**Comment by jgallimore** [comment]
Thanks for reporting this @edbratt . I have created a PR here for review: https://github.com/jakartaee/jekyll-theme-jakarta-ee/pull/16

**Comment by jgallimore** [comment]
Fixed in #16

**Comment by edbratt** [comment]
Sorry, found another case that causes the theme processing to halt:
/*
header a.button {
  background: transparent url(../images/logo.png) 0 0 no-repeat;
  padding-left: 32px;
}
*/

The e-mailed error is:

> Your SCSS file `guides/mq-dev-guide-java/assets/css/style.scss` has an error on line 32: Invalid CSS after "*/": expected identifier, was "header a.button". For more information, see https://help.github.com/articles/page-build-failed-invalid-sass-or-scss/.


I can work around this by just removing the offending code, but there are lots of documents I'd have to make that change in -- I presume this is just standard asciidoc boiler plate (though I could be mistaken)
